pclntab/pclntab.go

```diff
@@ -586,7 +587,7 @@ func findGoPCLnTab(ef *pfelf.File, additionalChecks bool) (goPCLnTabInfo *GoPCLn
 		// gopclntab parsing code might panic if the data is corrupt.
 		defer func() {
 			if r := recover(); r != nil {
-				err = fmt.Errorf("panic while searching pclntab: %v", r)
+				err = fmt.Errorf("panic while searching pclntab: %v, stack:\n%s", r, debug.Stack())
```

Comment:
Since failing to extract gopclntab is an info log, and not a debug log, I'd recommend against printing the stack here. I think customers would be confused seeing a fully printed stack in their info logs

Reply:
Removed stack printing